### ISSUE TITLE
Update To Test Structure Needed To Align With Upcoming R Version

### ISSUE THREAD

**Issue by bburns632** [UptakeOpenSource/pkgnet#160]
It appears the paths to test packages have issues with the latest CRAN Hosted r-devel on all platforms.  Error logs can be seen [HERE](https://cran.r-project.org/web/checks/check_results_pkgnet.html).

The R-devel version used to submit v0.3.0 was from December.  

Version of R-devel: 
- using R Under development (unstable) (2019-01-05 r75949)

 CRAN test version info [HERE](https://cran.r-project.org/web/checks/check_flavors.html#r-devel-linux-x86_64-debian-gcc)

Issue in Log (one example):
https://cran.r-project.org/web/checks/check_results_pkgnet.html
```Warning: invalid package ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/pkgnet.Rcheck/00_pkg_src/pkgnet//baseballstats’
     Warning: invalid package ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/pkgnet.Rcheck/00_pkg_src/pkgnet//sartre’
     Warning: invalid package ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/pkgnet.Rcheck/00_pkg_src/pkgnet//milne’
     Warning: invalid package ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/pkgnet.Rcheck/00_pkg_src/pkgnet’
     Error: ERROR: no packages specified
     Warning: invalid package ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/pkgnet.Rcheck/00_pkg_src/pkgnet//baseballstats’
     Warning: invalid package ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/pkgnet.Rcheck/00_pkg_src/pkgnet//sartre’
     Warning: invalid package ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/pkgnet.Rcheck/00_pkg_src/pkgnet//milne’
     Warning: invalid package ‘/home/hornik/tmp/R.check/r-devel-clang/Work/PKGS/pkgnet.Rcheck/00_pkg_src/pkgnet’
     Error: ERROR: no packages specified
     Error in pkgnet:::.BuildTestLib(targetLibPath = Sys.getenv("PKGNET_TEST_LIB")) : 
     Installation of packages in .BuildTestLib failed! (exit code = 1)```

**Comment by bburns632** [comment]
The changes introduced in #162 should correct these errors incurred on CRAN's testing infrastructure.  These errors stem from the inherit need of this package to test itself on toy packages of a known structure. We have gone to great lengths to remain in compliance with all CRAN policies by conduct testing within a temporary directory (#108).  However, despite no issues locally or via the automated TRAVIS checks of pull requests, the creation and later referencing of temporary directories on CRAN's testing infrastructure remains troublesome. 

As unit testing is at the discretion of the package authors, we choose to have TRAVIS remain the main vehicle for automated unit testing for this package.  Of course, all other tests of package integrity on CRAN will continue, just not custom unit testing.

**Comment by jayqi** [comment]
Resolved by #162.
